Gems/Multiplayer/Code/Source/NetworkEntity/NetworkEntityManager.cpp

```diff
@@ -467,6 +467,54 @@ namespace Multiplayer
         return netEntityId;
     }
 
+    AzFramework::EntitySpawnTicket NetworkEntityManager::RequestNetSpawnableInstantiation(const AZ::Data::Asset<AzFramework::Spawnable>& rootSpawnable)
+    {
+        const AzFramework::Spawnable::EntityList& entityList = rootSpawnable->GetEntities();
+        if (entityList.size() == 0)
+        {
+            AZ_Assert(false, "RequestNetSpawnableInstantiation: No entities in the spawnable %s", rootSpawnable.GetHint().c_str())
+            return {};
+        }
+
+        // The first entity in every spawnable is the root one 
+        const AZ::Entity* rootEntity = (entityList.begin())->get();
```

Comment:
Perhaps it's worth checking `rootEntity` pointer is valid in case the entity is invalid.

Reply:
Not sure if it is possible to null entities unless the app runs out of memory (which would be a fatal error), but added a check just in case!